src/codemirrorCommands.ts

```diff
@@ -8,16 +8,26 @@ import { CommandRegistry } from '@lumino/commands';
  */
 const IS_MAC = !!navigator.platform.match(/Mac/i);
 
+export interface Keybinding {
```

Comment:
```suggestion
export interface IKeybinding {
```

also needs a lint pass.

Reply:
Thanks, I've gotten so used to having this run as part of `yarn watch` I didn't realize I was missing it.